### PR TITLE
Constrain navbar width to 1780px

### PR DESCRIPTION
**Pull request by torztomasz**
Resolves L2B-743

### CODE REVIEW

File: packages/frontend/src/components/navbar/Navbar.tsx [hunk 1]
```diff
@@ -57,97 +57,94 @@ export function Navbar(props: NavbarProps) {
...
-          <MenuOpenIcon className="block" aria-label="Open menu" />
-        </button>
-        <ul className="flex h-full items-center">
-          <li
-            className={cx(
-              props.showBridges
-                ? 'mx-4 lg:ml-0 lg:mr-8'
-                : 'absolute left-1/2 top-1/2 -translate-x-1/2 -translate-y-1/2',
+      <nav className="lg:h-16border-b h-14 border-gray-200 text-base dark:border-gray-850">
+        <div className="relative mx-auto box-border flex h-full max-w-[1780px] items-center px-4 lg:justify-between lg:px-12">
```
Comment (michalsidzej) [inline packages/frontend/src/components/navbar/Navbar.tsx:61]: Nit pick: I would swap `div` and `nav` elements

File: packages/frontend/src/components/navbar/Navbar.tsx [hunk 2]
```diff
@@ -57,97 +57,94 @@ export function Navbar(props: NavbarProps) {
...
-        <button id="sidebar-menu-open" className="block lg:hidden">
-          <MenuOpenIcon className="block" aria-label="Open menu" />
-        </button>
-        <ul className="flex h-full items-center">
-          <li
-            className={cx(
-              props.showBridges
-                ? 'mx-4 lg:ml-0 lg:mr-8'
-                : 'absolute left-1/2 top-1/2 -translate-x-1/2 -translate-y-1/2',
+      <nav className="lg:h-16border-b h-14 border-gray-200 text-base dark:border-gray-850">
```
Comment (michalsidzej) [inline packages/frontend/src/components/navbar/Navbar.tsx:60]:
```suggestion
      <nav className="lg:h-16 border-b h-14 border-gray-200 text-base dark:border-gray-850">
```

**Review by michalsidzej — CHANGES_REQUESTED** [review]

File: packages/frontend/src/components/navbar/Navbar.tsx (hunk 1 above)
Comment (torztomasz) [reply packages/frontend/src/components/navbar/Navbar.tsx]: nice catch!

**Review by michalsidzej — APPROVED** [review]
LGTM

**Pull request merged**